scripts/maven/Dependency.kt

```diff
@@ -0,0 +1,19 @@
+import com.squareup.moshi.Json
```

Comment:
See my doc comment, but these classes should be organized in a package structure similar to #3340.

Reply:
Will use the same structure when script is configured properly.